src/main/webapp/script.js

```diff
@@ -20,16 +20,16 @@ google.charts.load('current', {
 
 function passQuery() {
   document.getElementById('map-title').innerText = '';
+  document.getElementById('more-info').innerText = 'Please wait. Loading...';
   document.getElementById('result').style.display = 'block';
-  const information = document.getElementById('more-info');
-  information.innerText = 'Please wait. Loading...';
 
   const query = new FormData(document.getElementById('query-form'));
   const personType = query.get('person-type');
   const action = query.get('action');
 
   const locationName = query.get('location');
-  const location = document.querySelector("#location option[value='"+locationName+"']").dataset.value;
+  const location = document.querySelector(
+    "#location option[value='"+locationName+"']").dataset.value;
```

Comment:
Single quotes to match style guide

Reply:
I think it has to be double quotes because I want the string that's created to have single quotes inside of it (the value='locationName' part)

Reply:
I think you can use single quotes in a single quote string with a backslash: '#location option[value=\''+locationName+'\']'

Reply:
That worked, thanks!